### ISSUE TITLE
sum instead of mean?

### ISSUE THREAD

**Issue by AngledLuffa** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#16]
Would you be up for adding a `reduction=...` option which allows for sum vs mean?
